js/views/view_logic.js

```diff
@@ -81,6 +81,6 @@ export function lastPage(){
 }
 
 export async function pauseAndContinue(){
```

Comment:
Why do we pause execution before proceeding in the app when the user clicks the next button?

Reply:
I was trying to smooth the transition between pages, but it doesn't really look as intended, so I'll delete.